projects/packages/sync/src/class-replicastore.php

```diff
@@ -1183,6 +1184,8 @@ public function checksum_all( $perform_text_conversion = false ) {
 		$terms_checksum              = $this->checksum_histogram( 'terms', null, null, null, null, true, '', false, false, $perform_text_conversion );
 		$term_relationships_checksum = $this->checksum_histogram( 'term_relationships', null, null, null, null, true, '', false, false, $perform_text_conversion );
 		$term_taxonomy_checksum      = $this->checksum_histogram( 'term_taxonomy', null, null, null, null, true, '', false, false, $perform_text_conversion );
+		$users_checksum              = $this->checksum_histogram( 'users', null, null, null, null, true, '', false, false, $perform_text_conversion );
```

Comment:
drop users and usermeta from the checksum_all. 
User meta has the potential to be a performance hit, as such we are going to call them on demand from WP.com this way we can enable/disable based on our testing/schedule without the need for a point release if there is a concern. If all is good we'll re-add this back in 10.3

Danger Will Robinson, Danger

Reply:
implemented.